llama_index/embeddings/openai.py

```diff
@@ -1,12 +1,16 @@
 """OpenAI embeddings file."""
 
 from enum import Enum
-from typing import Any, Dict, List, Optional, Tuple
+from typing import Any, Dict, List, Optional, Tuple, Callable
 
 import openai
 from tenacity import retry, stop_after_attempt, wait_random_exponential
 
 from llama_index.embeddings.base import BaseEmbedding
+from llama_index.callbacks.base import CallbackManager
+
+# Passed to the base embedding class
+DEFAULT_EMBED_BATCH_SIZE = 10
```

Comment:
nit: you can import `DEFAULT_EMBED_BATCH_SIZE` from `llama_index/embeddings/base.py` to avoid duplicating the default value.